Source/ResourceLibUnitTests/AcceleratorTests.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using Vestris.ResourceLib;
+
+namespace Res_sample

```

Comment:
This isn't a test, it's a sample. Run NUnit tests on this project, and extend https://github.com/dblock/resourcelib/blob/master/Source/ResourceLibUnitTests/AcceleratorResourceTests.cs with additional tests for various simple scenarios that you're implementing. There're lots of examples there.